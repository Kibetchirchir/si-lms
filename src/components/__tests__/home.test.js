import React from 'react';
import { shallow } from 'enzyme';

import HomePage from '../home';

describe('render profile View Container', () => {
    const wrapper = shallow(<HomePage />);
    it('renders profile component correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
