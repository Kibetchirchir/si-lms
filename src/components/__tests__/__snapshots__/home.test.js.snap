// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render profile View Container renders profile component correctly 1`] = `
<div>
  <withRouter(Connect(LoginForm)) />
</div>
`;
